Source/Utils/WarpXUtil.cpp

```diff
@@ -164,3 +164,47 @@ namespace WarpXUtilIO{
     }
 }
 
+void Store_parserString(std::string query_string,
+                        std::string& stored_string)
+{
+
+    ParmParse pp("warpx");
```

Comment:
Because of the line above, this version can only read input parameters with the form
```c++
warpx.<a function> = <a string>
```
Could we add an argument `parmparse_string` so that we could read
```c++
<an object>.<a function> = <a string>
```
?

Reply:
Done.